targets/targets.json

```diff
@@ -3132,7 +3132,7 @@
         "device_name": "NUC472HI8AE",
         "bootloader_supported": true
     },
-    "NCS36510": {
+    "36510": {
```

Comment:
I agree.  This looks like an unintentional change.  It causes everything to break.  

@jacobjohnson-ON Please push an additional commit on your branch that sets the target name back to "NCS36510".